test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -54,6 +58,42 @@ public async Task PriorityQueueModuleToModuleMessages()
             await this.ValidateResultsAsync();
         }
 
+        [Test]
+        public async Task PriorityQueueModuleToHubMessages()
+        {
+            // TODO: Add Windows and ARM32. Windows won't be able to work for this test until we add NetworkController Windows implementation
+            if (OsPlatform.IsWindows() || !OsPlatform.Is64Bit())
+            {
+                Assert.Ignore("Priority Queue module to module messages test has been disabled for Windows and Arm32 until we can fix it.");
+            }
+
+            CancellationToken token = this.TestToken;
+            string trcImage = Context.Current.TestResultCoordinatorImage.Expect(() => new ArgumentException("testResultCoordinatorImage parameter is required for Priority Queues test"));
+            string loadGenImage = Context.Current.LoadGenImage.Expect(() => new ArgumentException("loadGenImage parameter is required for Priority Queues test"));
+            string relayerImage = Context.Current.RelayerImage.Expect(() => new ArgumentException("relayerImage parameter is required for Priority Queues test"));
+            string networkControllerImage = Context.Current.NetworkControllerImage.Expect(() => new ArgumentException("networkControllerImage parameter is required for Priority Queues test"));
+            string trackingId = Guid.NewGuid().ToString();
+            TestInfo testInfo = this.InitTestInfo(5, 1000, true);
+
+            var testResultReportingClient = new TestResultReportingClient { BaseUrl = "http://localhost:5001" };
+
+            Action<EdgeConfigBuilder> addInitialConfig = this.BuildAddInitialConfig(trackingId, "hubtest", trcImage, loadGenImage, testInfo, true);
+            Action<EdgeConfigBuilder> addNetworkControllerConfig = this.BuildAddNetworkControllerConfig(trackingId, networkControllerImage);
+            EdgeDeployment deployment = await this.runtime.DeployConfigurationAsync(addInitialConfig + addNetworkControllerConfig, token);
+            await this.ToggleConnectivity(false, NetworkControllerModuleName, token);
```

Comment:
It's not easy to understand what false/true means for ToggleConnectivity. Maybe sending networkOn/Off or online/offline is more clear.